src/rest_framework_guardian/serializers.py

```diff
@@ -0,0 +1,86 @@
+from collections import Mapping
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.db import transaction
+from rest_framework.serializers import Serializer
+
+
+User = get_user_model()
+
+
+class DjangoGuardianObjectPermissionsAssigner(Serializer):
+    """
+    A serializer mixin that provides an easy way to assign permissions
+    to given users and/or group when an object is created or updated.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(DjangoGuardianObjectPermissionsAssigner, self).__init__(
```

Comment:
We're Python 3-only, so you can simplify the super call to `super().__init__(*args, **kwargs)`.